libs/gl-client/src/signer/mod.rs

```diff
@@ -679,7 +679,6 @@ mod tests {
                     signer_state: vec![],
                     requests: Vec::new(),
                 },
-                vec![],
```

Comment:
Removing this line since it's preventing the tests from building.

The method signature change was causing it to fail:
https://github.com/Blockstream/greenlight/pull/227/files#diff-8d5abc718d7a05d6432918246fcf5f9919681ca385b057f72a2117e9878e1932R284